golang-observ-lib/README.md

```diff
@@ -0,0 +1,10 @@
+# Golang observ lib
```

Comment:
Could you include some screenshots like the ones used in the PR body? I think that'd be great as an idea for people thinking of adopting it

Reply:
Added screenshots